Switch to the most recent Kibana configuration format and SAML/OIDC endpoints. (#50652)
elastic/elasticsearch: x-pack/docs/en/security/authentication/saml-guide.asciidoc

[source, yaml]
------------------------------------------------------------
xpack.security.authc.providers:
  saml.saml1:
    order: 0
    realm: "saml1"
  basic.basic1:
    order: 1
------------------------------------------------------------

> If {kib} is configured in this way, then users will be presented with the choice

Reviewer: Minor edit:
```suggestion
If {kib} is configured in this way, users are presented with a choice
```

> at the Login Selector UI. They can provide a username and password, and rely on one

Reviewer: Trying to make it clear that "in this sentence we're describing just two options - either SAML or not-SAML":
```suggestion
at the Login Selector UI. They log in with SAML or they provide a username and password and rely on one
```

Author: Thanks! Committed your suggestions.

> at the Login Selector screen whether to use username and password, and rely on one

Reviewer: ```suggestion
at the Login Selector screen. They can provide a username and password, rely on one
```

Author: Thanks! I kept `and` before `rely on` since these "other security realms" are used only when username and password are provided, but let me know if there is a better way to express this (in this sentence we're describing just two options - either SAML or not-SAML).

Reviewer: Hmmm... maybe that will be more obvious if we change the order.  I'll provide a new suggestion.

> of the  other security realms within {es}, or log in with SAML. Only users who have

Reviewer: Finishing the edit to this sentence:
```suggestion
of the  other security realms within {es}. Only users who have
```

> a username and password for a configured {es} authentication realm will be able to

Reviewer: Minor edit, removing future tense:
```suggestion
a username and password for a configured {es} authentication realm can
```
